.gitignore

```
@@ -1,3 +1,6 @@
 *.*~
+.DS_Store
+
 node_modules
+
 /sanscript.js
```

Why do we need to keep the `dist` directory in the repository to run tests? One can run `npm run dist` followed by `npm run test`.
```suggestion
/dist/
```
(if we proceed with a separate `dist` directory; see my comments above regarding that)

---

I'm getting rid of the dist folder after all.